events/lambda_http.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+package events
+
+// LambdaHTTPRequest contains data coming from the new HTTP API Gateway
+type LambdaHTTPRequest struct {
```

Comment:
I think I'm gonna change this to LambdaFunctionURLRequest/Response to align with the feature name in the documentation